src/testing/anndata/_pytest.py

```diff
@@ -11,17 +11,22 @@
 
 import re
 import warnings
+from importlib.metadata import version
 from importlib.util import find_spec
 from typing import TYPE_CHECKING, cast
 
+import pandas as pd
 import pytest
+from packaging.version import Version
 
 if TYPE_CHECKING:
     from collections.abc import Generator, Iterable, Sequence
     from pathlib import Path
 
     from ._doctest import WarningFilter
 
+# Use a marker present in the environment so VS Code’s tests behave identical
+IS_PRE = Version(version("zarr")).is_prerelease
```

Comment:
Yeah, I had no better idea. I want this option tied to the `pre` env, without relying on anything that can’t be inferred when running the tests through VS Code (i.e. no env vars, no CLI options)